### PR TITLE
add new nlist operator

### PR DESCRIPTION
**Pull request by sensorario**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Add an operator to exclude some values from results. Is the opposite of `list` operator.

### CODE REVIEW

File: src/Mado/QueryBundle/Queries/QueryBuilderFactory.php [hunk 1]
```diff
@@ -326,6 +336,7 @@ private function applyFilterOr($filter, $value, $orCondition)
     {
         $whereCondition = null;
         $filterAndOperator = explode('|',$filter);
+        $op = Objects\Operator::fromString($filterAndOperator[1]);
```
Comment (AlessandroMinoccheri) [inline src/Mado/QueryBundle/Queries/QueryBuilderFactory.php]: can we use only Operator::fromString($filterAndOperator[1]); without all path?

Comment (sensorario) [reply src/Mado/QueryBundle/Queries/QueryBuilderFactory.php]: 👍

File: src/Mado/QueryBundle/Queries/Objects/Operator.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Mado\QueryBundle\Queries\Objects;
```
Comment (AlessandroMinoccheri) [inline src/Mado/QueryBundle/Queries/Objects/Operator.php]: Can we merge this object with Dictionary/Operators or create new one?

Comment (sensorario) [reply src/Mado/QueryBundle/Queries/Objects/Operator.php]: I'll remove Dictionary\Operators and I'll keep only `Mado\QueryBundle\Dictionary`.

Comment (AlessandroMinoccheri) [reply src/Mado/QueryBundle/Queries/Objects/Operator.php]: 👍

**Pull request merged**